### ISSUE TITLE
pg_dump missing table data after starting the container

### ISSUE THREAD

**Issue by stevenjackson** [appropriate/docker-postgis#38]
We've been trying to backup a populated postgis database using pg_dump and haven't had much success.

So we went back to the simplest Dockerfile

```
FROM mdillon/postgis
```

After starting the container, we immediately bash'd in and ran pg_dump:

```
pg_dump -Upostgres -c postgres > dump
```

The dump file doesn't contain most of the useful seed tables, like `state_lookup`

We've tried many variations using `pg_dumpall`, specifying the schema using `-n tiger`, trying to grab specific tables, none have been successful.

Obviously this isn't a huge issue until we use the `loader_generate` scripts to pull in several download hours worth of data.  How do we back up the database inside this container?

**Comment by md5** [comment]
I can confirm this behavior and I'm not sure what's going on. I can see that the `tiger.state_lookup` and other tables exist in a newly created instance of this image, but I also don't see it dumped in `pg_dump`.

**Comment by md5** [comment]
Have you by any chance tested this outside of a Docker context?

**Comment by md5** [comment]
I just tested this using PostGIS 2.2.2 and PostgreSQL 9.5.4 on my Mac from Homebrew and I got the same results.

From a bit of searching around, I believe the issue is that tables that are owned by an extension are not dumped by `pg_dump` since the extension is expected to recreate them. The `pg_dump` output does include the relevant `CREATE EXTENSION` statements from what I can see, both in the Docker case and the Homebrew case (in fact the dumps are functionally identical).
